docs/concepts/html/elements.md

```diff
@@ -244,3 +244,102 @@ impl Component for MyComponent {
 }
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## Event Types
+
+| Event name  | `web_sys` Event Type | `stdweb` Event Type |
+| ----------- | -------------------- |  ------------------ |
+| `onabort` | [Event](https://docs.rs/web-sys/latest/web_sys/struct.Event.html) | [ResourceAbortEvent](https://docs.rs/stdweb/latest/stdweb/web/event/struct.ResourceAbortEvent.html) |
+| `onauxclick` | [MouseEvent](https://docs.rs/web-sys/latest/web_sys/struct.MouseEvent.html) | [AuxClickEvent](https://docs.rs/stdweb/latest/stdweb/web/event/struct.AuxClickEvent.html) |
+| `onblur` | [FocusEvent](https://docs.rs/web-sys/latest/web_sys/struct.FocusEvent.html) | [BlurEvent](https://docs.rs/stdweb/latest/stdweb/web/event/struct.BlurEvent.html) |
+| `oncancel` | [Event](https://docs.rs/web-sys/latest/web_sys/struct.Event.html) | Unsupported |
+| `oncanplay` | [Event](https://docs.rs/web-sys/latest/web_sys/struct.Event.html) | Unsupported |
+| `oncanplaythrough` | [Event](https://docs.rs/web-sys/latest/web_sys/struct.Event.html) | Unsupported |
+| `onchange` | [ChangeData](https://docs.rs/web-sys/latest/web_sys/struct.ChangeData.html) | [ChangeEvent](https://docs.rs/stdweb/latest/stdweb/web/event/struct.ChangeEvent.html) |
```

Comment:
`ChangeData` isn't defined in `web-sys`. Yew uses a special type for this: <https://docs.rs/yew/latest/yew/events/enum.ChangeData.html>.

Reply:
Fixed it